src/scss/2021/components/_form.scss

```diff
@@ -0,0 +1,42 @@
+.cmp-form-field {
+  &__input {
+    box-sizing: border-box;
+    width: 100%;
+    color: var(--color-neutral-1);
+    border: 1px solid var(--color-neutral-1);
+    line-height: 1.5;
+    padding: 0.75rem;
+    appearance: none;
+    border-radius: 0;
+    background: var(--color-neutral-3);
+    border-left-width: 0.25rem;
+
+    &:hover, 
+    &:focus {
+      border: 2px solid var(--color-neutral-1);
```

Comment:
This change on the border causes a flow change, meaning the content below it shifts down the size of the added border width. Instead of changing border sizes here, change the appearance of the border with a box-shadow or outline property.

Reply:
ahh ok!